walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -48,13 +48,27 @@ else if (request.getDuration() > TimeRange.WHOLE_DAY.duration()){
             TimeRange.fromStartEnd(event.getWhen().end(), TimeRange.END_OF_DAY, true));
     }
 
-    //fourth, fifth, sixth tests
+    //fourth, fifth, sixth, seventh (automatically covers double booking) tests
     for (Event event1 : events){
         for (Event event2 : events){
             if (event1 != event2){
 
                 //every attendee considered -- fourth test
                 if (!event1.getWhen().overlaps(event2.getWhen())){
+
+                    //just enough room
+                    if (event1.getAttendees() == event2.getAttendees()){
+                        if(event2.getWhen().start()-event1.getWhen().end() >= request.getDuration()){
+                            // ****Compilation failure
+                            //[ERROR] 
+                            ///home/talsternberg/step/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java
+                            //incompatible types: possible lossy conversion from long to int 
+                            //-- solvable or go about this a different way?
+                            availibility =
+                            Arrays.asList(TimeRange.fromStartDuration(event1.getWhen().end(), request.getDuration())); // **** error from here
```

Comment:
Compilation failure
[ERROR] /home/talsternberg/step/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:[68,114] incompatible types: possible lossy conversion from long to int

Reply:
Any advice for how to go about modifying this?

Reply:
`request.getDuration()` returns a long.  But method `fromStartDuration` is expecting an int.  A couple things you can do:
1.  change MeetingRequest duration to an int.
2.  explicitly cast form long to int before calling `fromStartDuration`